unpackaged/config/trial/flexipages/NPSP_Account_Record_Page.flexipage

```diff
@@ -270,16 +270,6 @@
         <name>facet-bba4c430-44a0-423e-8e7d-a0a6e28ee13b</name>
         <type>Facet</type>
     </flexiPageRegions>
-    <flexiPageRegions>
```

Comment:
I don't think we need to - or should - remove these from the unpackaged metadata. This will be used to configure trial orgs, where we want the component to show. It will not be pushed to existing customers, which is where the current issue is.

Reply:
That's fair, I can revert those, I got a little delete happy with this one.